language/tools/resource-viewer/src/lib.rs

```diff
@@ -226,6 +226,69 @@ fn pretty_print_ability_modifiers(f: &mut Formatter, abilities: AbilitySet) -> s
     Ok(())
 }
 
+impl serde::Serialize for AnnotatedMoveStruct {
+    fn serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        if serializer.is_human_readable() {
+            let mut s = serializer.serialize_map(Some(self.value.len()))?;
+            for (f, v) in &self.value {
+                s.serialize_entry(f, v)?
+            }
+            s.end()
+        } else {
```

Comment:
do we have something use the following part code? assuming get_resources used human readable part.

Reply:
Good point; I think this is unused. Will eliminate.